docs/examples/geopackage-json-ld.md

```diff
@@ -0,0 +1,118 @@
+# Schema.org Example Collection of Geopackage's
+
+Example metadata listing geopackage files using the [Schema.org](https://schema.org) vocabulary in
+`json+ld` format. See [#3](https://github.com/I-GUIDE/data-catalog/issues/3) and
+[#4](https://github.com/I-GUIDE/data-catalog/issues/4), for discussion that led to the creation of
+this example.
+
+```json
+{
+    "@context": "https://schema.org/",
+    "@type": "Dataset",
+    "url": "https://www.hydroshare.org/resource/3295a17b4cc24d34bd6a5c5aaf753c50",
+    "name": "geoconnex reference features",
+    "description": "These GeoPackage artifacts are for use in https://reference.geoconnex.us For more information see: https://github.com/internetofwater/geoconnex.us/tree/master/namespaces/ref/",
+    "creator": [
+        {
+            "@type": "Organization",
+            "email": "cgsinternetofwater@gmail.com",
+            "name": "Internet of Water Initiative, Center for Geospatial Solutions, Lincoln Institute of Land Policy",
+        }
+    ],
+    "provider": {
+        "@id": "https://www.hydroshare.org",
+        "@type": "Organization",
+        "name": "HydroShare",
+        "url": "https://www.hydroshare.org"
+    },
+    "distribution": {
+        "@type": "DataDownload",
+        "contentSize": "3.5 GB",
+        "encodingFormat": "application/zip",
+        "contentUrl": "https://www.hydroshare.org/hsapi/resource/3295a17b4cc24d34bd6a5c5aaf753c50/",
+        "description": "Zipped BagIt Bag containing the HydroShare Resource",
+        "dateModified": "2022-08-03T15:41:57.592393+00:00",
+        "identifier": [
+            "https://www.hydroshare.org/resource/3295a17b4cc24d34bd6a5c5aaf753c50"
+        ]
+    },
+    "hasPart": [
+        {
+            "@type": "Dataset",
+            "isPartOf": "https://www.hydroshare.org/resource/3295a17b4cc24d34bd6a5c5aaf753c50",
+            "associatedMedia": {
+                "@type": "MediaObject",
+                "contentSize": "382255104",
```

Comment:
Should content size have a unit label associated with it? According to SchemaOrg it represents "File size in (mega/kilo) bytes."

Reply:
I don't think this comment needs to hold up the PR merge so I created a ticket to continue the discussion: #7